src/bft.py

```diff
@@ -12,4 +12,16 @@ def bf_order(t: T | None) -> Iterable[int]:
     >>> list(bf_order(tree))
     [2, 1, 4, 3, 5]
     """
-    return  # FIXME
+    queue = deque()
+    output = []
+    if not t:
+        return []
+    queue.append(t)
+    while len(queue) > 0:
```

Comment:
You can also just write

```python
while queue:
    ...
```

That would be the more Pythonic way of testing if a data structure is empty.